lib/drivers/onetouch/oneTouchUltra2.js

```diff
@@ -401,7 +385,26 @@ module.exports = function (config) {
       };
 
       async.retry({ times: RETRIES, interval: TIME_TO_WAIT }, sendReceive, function(err, result) {
-        callback(err, result);
+
+        common.checkDeviceTime(cfg, function(err, serverTime) {
+          if (err == 'updateTime') {
+            cfg.deviceInfo.annotations = 'wrong-device-time';
+            debug('Time set to');
```

Comment:
Either remove debug statement or add time?

Reply:
That line confused me too, until I ran the code and realised that it's printing the time on the next line (in the `setDateTime` method).